### DISCUSSION TITLE
loading the arrow files

### DISCUSSION THREAD

**Discussion by tanasa** [GreenleafLab/ArchR#1430] [General]
Hi Ryan, 

Would you please let me know : can we load an Arrow file in R if we do not have all the ArchR project files ?

Thank you very much, 

~ Bogdan

**Comment by rcorces** [comment]
This is a complicated question because ArrowFiles become inherently linked to an ArchRProject object as they get updated and matrices are added etc.

If your ArrowFile just has fragment information / other information created during ArrowFile creation, then it can be loaded without an ArchRProject. But if you have added different matrices etc, then you would not want to load those ArrowFiles without the associated ArchRProject.

**Comment by tanasa** [comment]
Thank you Ryan. Apparently the shiny application before starts from the arrow files ? 

https://cancell.medisin.uio.no/ShinyArchR.UiO/

**Reply by paupaiz** [reply]
We are working on a version of the Shiny app that will not require arrow files.

**Reply by rcorces** [reply]
@tanasa - I dont think that is correct. I believe that ShinyArchRUiO uses the ArchRProject
https://github.com/EskelandLab/ShinyArchRUiO#:~:text=Provide%20the%20path,path%20to%20projHeme5/%22)

**Reply by tanasa** [reply]
Hi Ryan, yes, you are right. In the R script, the authors specify : 

`savedArchRProject1 <- loadArchRProject("~/ArchRAnalysis/Save-ProjHeme2/")`
